static/css/components/work.less

```
@@ -50,6 +51,12 @@ div.editionAbout {
   margin-top: 0;
 }
 
+.work-description{
+  pre {
+    overflow: auto;
+  }    
```

```suggestion
  }
```
Our linting rules do not allow whitespace at the end of a line.  This is causing our Javascript tests to fail.

You can run the JS and CSS linter locally by running the following:
`docker-compose exec -uroot web npm run lint`

---

Okay. Will change that.